docs/v1/P4Runtime-Spec.mdk

```diff
@@ -3701,7 +3701,8 @@ An update can be one of the following types:
   `NOT_FOUND` error is returned. In order to delete, the entity specification
   only needs to include the key. Any non-key parts of entity are ignored.
 
-The `Write` RPC is idempotent, &ie; multiple invocations of the same RPC do not
+The `Write` RPC is idempotent, &ie; multiple invocations of the same RPC,
```

Comment:
could you fix the wrapping of this paragraph?

Reply:
My text editor seems to be set to a shorter line width for automatic text wrapping -- hopefully this is reasonable.